supabase/functions/_script/backup_and_cleanup.ts

```diff
@@ -0,0 +1,86 @@
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@^2.2.3';
+import 'https://deno.land/x/dotenv/load.ts';
+import { r2 } from '../_utils/r2';
+import type { Database } from '../_utils/supabase.types.ts';
+import * as fs from 'https://deno.land/std/fs/mod.ts';
+
+const supabaseUrl = Deno.env.get('SUPABASE_URL') || '***';
+const supabaseServiceRoleKey = Deno.env.get('SUPABASE_SERVICE_ROLE_KEY') || '***';
```

Comment:
if these are not present, the script shall exit and not be marked as "***", right?